### ISSUE TITLE
[VSCode][Accessibility]A11y_HDInsight Tools for VSCode_Azure HDInsights Tools(installation)_Keyboard: Rating link is inaccessible using keyboard

### ISSUE THREAD

**Issue by jingyanjingyan** [microsoft/vscode#76064]
- VSCode Version: 1.34.0​
- OS Version: Mac OS, windows 10

Environment Details:

https://marketplace.visualstudio.com/items?itemName=mshdinsight.azure-hdinsight

Visual studio code version:1.34.0​

Repro Steps: 

1. Download and install HDInsight's for VS code from the URL: https://marketplace.visualstudio.com/items?itemName=mshdinsight.azure-hdinsight

2.Verify that rating link is accessible using keyboard or not.

Actual Result: 

Rating link is inaccessible using keyboard

Expected Result: 

Rating link should be accessible using keyboard

User Impact: 

Keyboard only users may face inconvenience if an interactive control is inaccessible using keyboard. They won't be able to use it while navigating using keyboard.

MAS Reference:

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Does this issue occur when all extensions are disabled?: Yes/No

Attach the pic
![Bug_279_Link is inaccessible](https://user-images.githubusercontent.com/36756024/60069397-2a510080-9745-11e9-85a9-e7e49677cf05.png)

Reference issue from HDInsight team:
https://dev.azure.com/bigdatabj/HDInsightVSCodeExtension/_workitems/edit/279
